skema/skema-rs/skema/src/config.rs

```diff
@@ -1,36 +1,57 @@
+use neo4rs::*;
 use std::env;
-use rsmgclient::{ConnectParams};
+// NOTE: this is deprecated
```

Comment:
I think this can be removed, @Free-Quarks .  I had such a note on the methods associated with the rsmgclient connection.  Now that those methods have been removed/the migration to neo4rs is complete, this comment is no longer relevant.

Reply:
Sounds good! Done!

Thanks!